qqsort.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * quicker quicksort
+ * non-blocking, efficient
+ *
+ * 2015-10-18 - AR.
+ */
+
+'use strict'
+
+var nextTick = global.setImmediate || process.nextTick
+
+module.exports = function qqsort( array, compar, callback ) {
```

Comment:
Can you gracefully handle the error case where someone just calls `qqsort()` without passing any arguments?